src/typing/flow_error.ml

```diff
@@ -165,6 +165,7 @@ type error_message =
       null_loc: Loc.t;
       falsy_loc: Loc.t;
     }
+  | EUnknownProperty of Loc.t * string
```

Comment:
Could we rename this error to `NonstrictLookupFailed`? This would be symmetrical with our error for strict lookup failures: [`EStrictLookupFailed`](https://github.com/facebook/flow/blob/dab35ec2fe38a83f2e6a25a5ba8aa082a26ef902/src/typing/flow_js.ml#L5471-L5476) (you might also want to look at [the message for `EStrictLookupFailed`](https://github.com/facebook/flow/blob/dab35ec2fe38a83f2e6a25a5ba8aa082a26ef902/src/typing/flow_error.ml#L773-L791)).